### PR TITLE
Group 77 api to delete post

### PR DESCRIPTION
**Pull request by balamurali0532**
Docker File and CI/CD yaml script changes

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FROM node:14-slim
+
+WORKDIR /usr/src/app
+
+COPY Backend/package*.json ./
+
+RUN npm install
+
+COPY Backend/. /usr/src/app/Backend/
```
Comment (udkreddy) [inline Dockerfile:9]: Copy whole application instead of just the backend

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+EXPOSE 3000
```
Comment (udkreddy) [inline Dockerfile:11]: No need to expose 3000 port
we will just expose the frontend port, backend can be kept internal

**Review by udkreddy — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (balamurali0532) [reply Dockerfile:9]: Handled in frontend docker file

File: Dockerfile (hunk 2 above)
Comment (balamurali0532) [reply Dockerfile:11]: For Backend image testing i just expose the port it will be removed in the final integration

**Review by udkreddy — APPROVED** [review]

**Pull request merged**